delivery/operator/README.md

```diff
@@ -0,0 +1,260 @@
+   
+An example follows. You may also want to review the test script at
+[./test/install-operator-framework.sh](./test/install-operator-framework.sh).
+
+```bash
+OSDK_VERSION=1.16.0
+DEST_DIR=/usr/local/bin
+OS=linux
+ARCH=amd64
+
+curl -L -o "${DEST_DIR}/operator-sdk" \
+    https://github.com/operator-framework/operator-sdk/releases/download/v${OSDK_VERSION}/operator-sdk_${OS}_${ARCH}
+chmod +x "${DEST_DIR}/operator-sdk"
+
+operator-sdk version
+operator-sdk olm install
+operator-sdk olm status
+```
+
+## Deliver
+
+Deliver podtato-head by creating an operator to manage "PodtatoHeadApp"
+resources, then declaring such a resource per the following instructions. You
+may also want to review the test script at
+[./test/build-and-push-operator.sh](./test/build-and-push-operator.sh).
+
+### Build a Helm chart-based operator
+
+In an empty directory init a new Helm operator from the podtato-head chart as follows.
+
+```bash
+chart_dir=./delivery/chart
+# expand relative path so it works from other dirs
+chart_dir=$(cd ${chart_dir} && pwd)
+
+work_dir=./work
+mkdir -p ${work_dir}
+cd ${work_dir}
+
+operator-sdk init --plugins 'helm.sdk.operatorframework.io/v1' --project-version=3 \
+    --project-name 'podtato-head-operator' \
+    --domain=podtato-head.io \
+    --group=apps \
+    --version=v1alpha1 \
+    --kind=PodtatoHeadApp \
+    --helm-chart=${chart_dir}
+```
+
+Now build and push the operator image, the operator bundle, and a catalog which
+includes the bundle as follows. We use GitHub's container registry `ghcr.io`;
+you will need to be signed in to push images. Sign in by getting a personal
+access token with `packages:write` permissions and running the following commands.
+
+```bash
+github_user=
+github_token=
+
+echo "${github_token}" | docker login --username=${github_user} --password-stdin ghcr.io
+
+export IMAGE_TAG_BASE=ghcr.io/${github_user}/podtato-head/operator
+export IMG=${IMAGE_TAG_BASE}:latest
+
+make docker-build docker-push
+make bundle bundle-build bundle-push
+make catalog-build catalog-push
+```
+
+### Deploy the operator
+
+To deploy the operator we'll deploy the catalog source and then subscribe to our
+package, as follows. You may also want to review the test script at
+[./test/deploy-operator.sh](./test/deploy-operator.sh).
+
+First deploy the catalog source to the OLM namespace:
+
+```bash
+image_base_url=ghcr.io/${github_user}/podtato-head/operator
+operator_version=0.0.1
+
+kubectl apply -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+    name: podtato-head-catalog
+    namespace: olm
+spec:
+    sourceType: grpc
+    image: ${image_base_url}-catalog:v${operator_version}
+    secrets:
+      - ghcr
+    displayName: podtato-head-catalog
+    publisher: podtato-head
+    updateStrategy:
+        registryPoll:
+            interval: 10m
+EOF
+```
+
+Next declare a subscription to the podtato-head-operator in the `operators` namespace:
```

Comment:
I prefer creating a subscription as you've done here, but I believe that the SDK [provides a command](https://sdk.operatorframework.io/docs/olm-integration/quickstart-bundle/) to create install an operator which may be worth considering.

Reply:
@awgreene on review I think you may have been referring to the `make deploy` target [mentioned below](https://github.com/podtato-head/podtato-head/pull/144#discussion_r800913922). There's a section later in the doc which talks about that option, I updated that section's title to make it clearer.

Thanks!